README.md

```diff
@@ -24,6 +24,14 @@ Preparing your environment variables:
 source ./test_ws/install/setup.bash
 source $HOME/bin/setup-opt-robotpkg.sh
 ```
+If your installation uses the binaries, you will need to install the following packages, on top of the required `robotpkg-sot-core-v3` and `robotpkg-py27-talos-dev`:
+```
+sudo apt install robotpkg-talos-simulation robotpkg-talos-metapkg-ros-control-sot robotpkg-py27-sot-talos-balance robotpkg-py27-sot-pattern-generator-v3 robotpkg-ros-ouster-gazebo-simulation
```

Comment:
This shouldn't be required. The point of the `robotpkg-py27-talos-dev` package, is that this package is nothing but comes with dependency on everything we need. `robotpkg-ros-ouster-gazebo-simulation` is not yet included, that's true, but I'd prefer to fix that, instead of documenting a workaround.

Reply:
Ok, should I add the necessary dependencies in `robotpkg-p27-talos-dev` directly then? And even though `robotpkg-ros-ouster-gazebo-simulation` is not in it yet, shouldn't at least `talos-simulation` for example already be in it?...

Reply:
`robotpkg-talos-simulation` is an indirect dependency of `robotpkg-py27-talos-dev`. So installing `robotpkg-py27-talos-dev` is a sufficient condition to get `robotpkg-talos-simulation`.

Reply:
I added these installations because they were missing after installing `robotpkg-sot-core-v3` and `robotpkg-py27-talos-dev`, which made the build fail. Perhaps `ouster_gazebo_simulation` was the only one really necessary on top of these two, I'll try again later.